### ISSUE TITLE
How do I open '.seb' files?

### ISSUE THREAD

**Issue by Fillinlak3** [quintenvandamme/unsafeexambrowser#3]
I'm trying to open a .seb config file, but it doesn't work. If I have SafeExamBrowser installed, the `.seb` file will be opened in SEB insead of unSEB. How can I open a configuration file in unSEB?

**Comment by twoelw** [comment]
Try dragging and dropping it into unsafeexambrowser

**Comment by Fillinlak3** [comment]
i tried but it doesn't work

**Comment by twoelw** [comment]
Do you get an error or does it not open completely? Is it password protected?

**Comment by Fillinlak3** [comment]
![image](https://github.com/user-attachments/assets/27e47c7b-79d5-4104-9d87-1438eb7246fb)
It looks like this

**Comment by twoelw** [comment]
Drag it to unsafeexambrowser directly and not the shortcut, open file location until you reach the .exe file

**Comment by Fillinlak3** [comment]
let me try.

**Comment by Fillinlak3** [comment]
yeah it worked, thanks.

**Comment by Fillinlak3** [comment]
Btw, does it log that I'm using another version or that I have other apps active in background? Also is there any way to mask the Win+Tab or Shift+Esc? What if the professor comes to my pc and asks to do one of the combinations? It should be a combination that locks the combinations for safety.

**Comment by twoelw** [comment]
I don’t think it does, but I don’t think there’s a way to lock the keyboard combinations. That’s why I personally use a VM over this so I can enable exclusive mode and stop the shortcuts if someone’s behind me lol

**Comment by Fillinlak3** [comment]
What VM are u using VMVB or VMWare?

**Comment by quintenvandamme** [comment]
> [edited] Btw, does it log that I'm using another version or that I have other apps active in background? Also is there any way to mask the Win+Tab or Shift+Esc? What if the professor comes to my pc and asks to do one of the combinations? It should be a combination that locks the combinations for safety. Also, disable the background if the window is minimized. Make it like in the real one, dark blue or smth like that.

Making the inputs to that of Seb won't work because unsafeexambrowser uses the Windows Windows manager and not Seb kiosk mode. The background changing to the Dael Blue one is a great idea!

**Comment by twoelw** [comment]
Used to use Virtualbox but I’m using vmware workstation now because all the features are nice (especially exclusive mode) and it can spoof hardware pretty easily to avoid logs looking weird, around 7 of my friends have been using it for a month and noones suspicious yet

**Comment by twoelw** [comment]
> > [edited] Btw, does it log that I'm using another version or that I have other apps active in background? Also is there any way to mask the Win+Tab or Shift+Esc? What if the professor comes to my pc and asks to do one of the combinations? It should be a combination that locks the combinations for safety. Also, disable the background if the window is minimized. Make it like in the real one, dark blue or smth like that.
> 
> Making the inputs to that of Seb won't work because unsafeexambrowser uses the Windows Windows manager and not Seb kiosk mode. The background changing to the Dael Blue one is a great idea!

the backgrounds black for me lol

**Comment by twoelw** [comment]
> > [edited] Btw, does it log that I'm using another version or that I have other apps active in background? Also is there any way to mask the Win+Tab or Shift+Esc? What if the professor comes to my pc and asks to do one of the combinations? It should be a combination that locks the combinations for safety. Also, disable the background if the window is minimized. Make it like in the real one, dark blue or smth like that.
> 
> Making the inputs to that of Seb won't work because unsafeexambrowser uses the Windows Windows manager and not Seb kiosk mode. The background changing to the Dael Blue one is a great idea!

Maybe a companion app should be created to enable some sort of “panic mode” when a combination is pressed to disable keyboard shortcuts or temporarily enable kiosk mode

**Comment by Fillinlak3** [comment]
yeah that's what I mean. It should have a combination that activates a panic mode where the background changes to the original dark blue and where u cant see any of the windows tabs opened in the background.

**Comment by twoelw** [comment]
But I think it’s impossible to implement it into SEB itself so it had to be a companion app I think

**Comment by quintenvandamme** [comment]
Would be a lot of work, i have made a panic mode before for a chatgpt website when i was still in school and they blocked it. You press on q and it show a wikipedia article. But ill see what i can do.
